gqueries/output_elements/output_series/bezier_65_source_of_hot_water_in_households/solar_used_for_hot_water_in_households.gql

```diff
@@ -4,5 +4,6 @@
 - query =
     SUM(
       V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_radiation),
-      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal)      
+      V(CHILDREN(V(households_useful_demand_for_hot_water_after_solar_heater_and_p2h)),input_of_solar_thermal),
```

Comment:
No, you get it 200% correctly. These lines are doing nothing at the moment. They also do no real harm (except confusing you 😉 ).